expfmt/text_parse_test.go

```diff
@@ -385,6 +385,159 @@ request_duration_microseconds_count 2693
 				},
 			},
 		},
+		// 5: UTF-8 counter
```

Comment:
Nit: This should end with a period to be consistent with the naming above.

More substantially: Let's be more explicit here about what's significant. It's not special that this is a counter, and "UTF-8" per se doesn't say anything (the old metric names were all valid UTF-8, too). How about "Quoted metric name and quoted label name with dots."?